modules/serverless-rn-1-7-0.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.7.0
+
+[id="new-features-1-7-0_{context}"]
+== New features
+* {ServerlessProductName} 1.7.0 is now Generally Available (GA) on {product-title} 4.3 and newer versions. In previous versions, {ServerlessProductName} was a Technology Preview.
+* {ServerlessProductName} now uses Knative Serving 0.13.2.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.2.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* Knative `kn` CLI downloads now support disconnected, or restricted network installations.
+* Knative `kn` CLI libraries are now signed by Red Hat.
+* Knative Eventing is now available as a Technology Preview with {ServerlessProductName}. {ServerlessProductName} uses Knative Eventing 0.13.2.
+
+[IMPORTANT]
+====
+Before upgrading to the latest Serverless release, you must remove the community Knative Eventing operator if you have previously installed it. Having the Knative Eventing operator installed will prevent you from being able to install the latest Technology Preview version of Knative Eventing that is included with {ServerlessProductName} 1.7.0.
+====
+
+* High availability (HA) is now enabled by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
++
+If you have installed a previous version of {ServerlessProductName}, after the KnativeServing custom resource (CR) is updated, the deployment will default to a HA configuration with `KnativeServing.spec.high-availability.replicas = 2`.
++
+You can disable HA for these components by completing the procedure in the _Configuring high availability components_ documentation.
+* {ServerlessProductName} now supports the `trustedCA` setting in {product-title}'s cluster-wide proxy, and is now fully compatible with {product-title}'s proxy settings.
+* {ServerlessProductName} now supports HTTPS using the wildcard certificate that is registered for {product-title} routes. For more information on `http` and `https` on Knative Serving, see the documentation on _Verifying your serverless application deployment_.
+
+[id="fixed-issues-1-7-0_{context}"]
+== Fixed issues
+* In previous versions, requesting KnativeServing custom resources (CRs) without specifying an API group, for example, `oc get knativeserving -n knative-serving`, occasionally caused errors. This issue is fixed in {ServerlessProductName} 1.7.0.
+
+[id="known-issues-1-7-0_{context}"]
+== Known issues
+* When upgrading from {ServerlessProductName} 1.6.0 to 1.7.0, support for HTTPS requires a change to the format of routes. This causes Knative services created on {ServerlessProductName} 1.6.0 to stop working.
```

Comment:
This is only partly true. The old URLs stop working, meaning the services won't be reachable under their former URLs anymore. The services themselves will continue working, but are only reachable under the URL in the new format.

Reply:
I'm wondering if we should add a note about the changed URL into the upgrade docs somewhere here: https://github.com/openshift/openshift-docs/pull/20909/files#diff-87d4ec29515b83873f31f3ce2bf3a6f0R27
The user essentially has to retrieve the new URL after upgrade.

Reply:
@mgencur can you provide details for how they would do this? Is this the same as what's in the https://docs.openshift.com/container-platform/4.3/serverless/knative_serving/interacting-serverless-apps.html#verifying-serverless-app-deployment_interacting-serverless-apps section re retrieving the URL?
Please provide details of exactly what needs to be included.

Reply:
Yes, exactly as the section you referenced.

Reply:
More precisely. I'd say just the point "Find the application URL."